### ISSUE TITLE
[Windows 11] running `build-figma-plugin --typecheck --watch` opens `cli.js` in editor instead of executing it with node

### ISSUE THREAD

**Issue by fwextensions** [yuanqing/create-figma-plugin#93]
Every time I type `num run watch`, it starts watching the source files but it also opens the `src` directory in a new instance of my editor, which is Webstorm.  It also opens `./node_modules/typed-css-modules/lib/cli.js` in the existing editor instance.  I've never seen this behavior with any other npm scripts.  

I doubt it matters, but here's my `package.json`.

<details>
  <summary>package.json</summary>

```
{
  "name": "lorem-fitem",
  "version": "0.0.1",
  "description": "Your Figma plugin",
  "author": "John Dunning",
  "license": "",
  "figma-plugin": {
    "editorType": [
      "figma"
    ],
    "id": "1080322942020337214",
    "name": "Lorem Fit'em",
    "main": "src/main/main.ts",
    "ui": "src/ui/ui.tsx",
    "relaunchButtons": {
      "update": {
        "name": "Update placeholder text",
        "main": "src/main/main.ts",
        "ui": "src/ui/ui.tsx",
        "multipleSelection": true
      },
      "randomize": {
        "name": "Randomize placeholder text",
        "main": "src/main/main.ts",
        "ui": "src/ui/ui.tsx",
        "multipleSelection": true
      }
    }
  },
  "dependencies": {
    "@create-figma-plugin/ui": "^1.8.4",
    "@create-figma-plugin/utilities": "^1.8.4",
    "debounce": "^1.2.1",
    "preact": "^10"
  },
  "devDependencies": {
    "@create-figma-plugin/build": "^1.8.4",
    "@create-figma-plugin/tsconfig": "^1.8.4",
    "@figma/plugin-typings": "*",
    "@types/debounce": "^1.2.1",
    "hyphen": "^1.6.4",
    "typescript": "*"
  },
  "scripts": {
    "postinstall": "npm run build:text",
    "build": "build-figma-plugin --typecheck --minify",
    "build:text": "node text/hyphenateText.js",
    "watch": "build-figma-plugin --typecheck --watch"
  }
}
```
</details>

**Comment by fwextensions** [comment]
@juarrow has found what's likely the source of this buggy behavior: https://github.com/yuanqing/create-figma-plugin/issues/96#issuecomment-1090707570

**Comment by yuanqing** [comment]
This should be fixed in `1.9.1`
